### DISCUSSION TITLE
Oracle to PostgreSql

### DISCUSSION THREAD

**Discussion by KrishnaprasadN** [osalvador/ReplicaDB#48] [General]
Hi,

We want to migrate the Oracle View data to postgreSql tables.  Instead of Tables, can we use View? Basically, the client just exposes the Views, no direct access to the tables. Is this possible. 

I checked the wizard setup, there i can see only table name or custom query option didnt find any option for View.

**Comment by osalvador** [comment]
Hi @KrishnaprasadN,

Yes, of course, the source tabla can be a view, a table or a custom query. In fact, a view is just a virtual table based on the result-set of a custom query. 

You can use the `source.table parameter` to define your source view. 

Regards!
